src/Faker/Provider/et_EE/Person.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Faker\Provider\et_EE;
+
+/**
+ * Class Person
+ * @package Faker\Provider\et_EE
+ * @author David Gegelija <code@imdavid.xyz>
+ */
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link https://news.err.ee/114745/most-popular-baby-names-of-2014
+     */
+    protected static $firstNameMale = array(
+        'Rasmus', 'Artjom', 'Robin', 'Martin', 'Oliver', 'Romet', 'Sebastian', 'Robert',
```

Comment:
This dataset is too small, please look for a (public) dataset of at least 100 names. Oh, and please order alphabetically, and split line by capital letter/